centreon/tests/e2e/features/Resources-status/03-downtime.feature

```diff
@@ -25,25 +25,22 @@ Feature: Add a downtime on a resource
     Then the user must be notified of the sending of the order
     And the user should see the downtime resources appear in the listing after a refresh
 
-# TO BE FIXED: it hangs on the "the user selects the checkboxes and clicks on the "Cancel" action" step
-  # @TEST_MON-22208
-  # Scenario: Cancel a downtime on a resource
-  #   Given a resource is in downtime
-  #   And that you have to go to the downtime page
-  #   When I search for the resource currently "In Downtime" in the list
-  #   Then the user selects the checkbox and clicks on the "Cancel" action
-  #   Then the user confirms the cancellation of the downtime
-  #   Then the line disappears from the listing
-  #   Then the user goes to the Resource Status page
-  #   And the resource should not be in Downtime anymore
+  @TEST_MON-22208
+  Scenario: Cancel a downtime on a resource
+    Given a resource is in downtime
+    And that you have to go to the downtime page
+    When I search for the resource currently "In Downtime" in the list
+    Then the user selects the checkbox and clicks on the "Cancel" action and confirms the cancellation
```

Comment:
there are too much details in this step, this should be reformulate to get the functional action
ex:
```
Then the user starts downtime configuration on the resource
And the user cancels the downtime configuration
```